javaslang/src/test/java/javaslang/collection/TraverableTest.java

```diff
@@ -0,0 +1,45 @@
+package javaslang.collection;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+
+public class TraverableTest {
+
+    // -- max()
+
+    @Test
+    public void maxShouldBeEmpty() {
+         assertTrue(Iterator.empty().max().isEmpty());
+    }
+
+    @Test
+    public void maxShouldBeTheOnlyValue() {
```

Comment:
Minor: The name could be aligned to the other test names in AbstractTraversableTest, e.g. `shouldComputeMaxOfOneValue`.